packages/edit-widgets/src/components/sidebar/index.js

```diff
@@ -118,7 +118,7 @@ export default function Sidebar() {
 			isGeneralSidebarOpen
 		) {
 			enableComplementaryArea(
-				editWidgetsStore,
+				'core/edit-widgets',
```

Comment:
I believe a better idea is to use `.name`.

```diff
-'core/edit-widgets',
+editWidgetsStore.name,
```

Reply:
In this context `'core/edit-widgets'` is the name of the complementary area and not the name of the store.